src/subcommands/init

```diff
@@ -25,7 +25,7 @@ cmd-init() {
   echo "$PREFIX Selecting network interface"
   ETH_LINK=$(get-network-interface-name)
 
-  NODE_IP=$(ip -o route show to default | awk '{print $3}')
+  NODE_IP=$(curl -L ipconfig.me)
```

Comment:
Thanks for the suggestion, that works.
It's not the most stable solution, to be dependant on the response of an external service for initializing the swarm - which is quite critical.

What is the output of this on Vultr?
```sh
hostname -I | awk '{print $1}'
```

Reply:
Yes, it does.

Reply:
Let's use that for the moment, agree?